tests/conftest.py

```diff
@@ -49,6 +49,8 @@
 class TestSearch(RecordsSearch):
     """Test record search."""
 
+    boosted_fields = ["title^5"]
```

Comment:
could this be a part of RECORDS_REST_CONFIG ? in this way is a bit hidden. ping @ntarocco